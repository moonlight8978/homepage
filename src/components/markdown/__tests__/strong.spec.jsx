import React from 'react'
import { render } from '@testing-library/react'

import Markdown from '../markdown'

test('render correctly', () => {
  const content = '**This is a bold text**'
  const { container } = render(<Markdown content={content} />)
  expect(container.firstChild).toMatchSnapshot()
})
