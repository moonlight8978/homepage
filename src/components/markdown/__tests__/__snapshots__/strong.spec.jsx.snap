// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div
  class="markdownBody"
>
  <div
    class="paragraph"
  >
    <strong
      class="strong"
      data-testid="markdown-strong"
    >
      This is a bold text
    </strong>
  </div>
</div>
`;
